src/grt/src/MakeWireParasitics.cpp

```diff
@@ -83,8 +83,18 @@ void MakeWireParasitics::estimateParasitcs(odb::dbNet* net,
   parasitic_
       = parasitics_->makeParasiticNetwork(sta_net_, false, analysis_point_);
   makePinRoutePts(pins);
-  makeRouteParasitics(net, routes);
-  makeParasiticsToGrid(pins);
+
+  int routes_cnt = 0;
+  makeWireParasitics(net, routes, routes_cnt);
```

Comment:
why not just return routes_cnt (more properly called route_cnt) instead of passing the reference?